src/glue/jobs/compare_parquet_datasets.py

```diff
@@ -280,10 +285,10 @@ def get_cohort_from_s3_uri(s3_uri: str) -> Union[str, None]:
         Union[str, None]: the cohort if it exists
     """
     cohort = None
-    if "adults_v1" in s3_uri:
-        cohort = "adults_v1"
-    elif "pediatric_v1" in s3_uri:
-        cohort = "pediatric_v1"
+    if COHORTS["adults"] in s3_uri:
```

Comment:
Could simplify this further instead of using a dict to map this:

```

ADULTS = "adults_v1"
PEDIATRIC = "pediatric_v1

    if ADULTS  in s3_uri:
        cohort = ADULTS 
    elif PEDIATRIC  in s3_uri:
        cohort = PEDIATRIC 
```